dataflow/src/main/java/org/checkerframework/dataflow/analysis/AnalysisResult.java

```diff
@@ -21,12 +21,13 @@
  * providing the abstract values given a node or a tree. Note that it does not keep track of custom
  * results computed by some analysis.
  *
- * @param <A> type of the abstract value that is tracked
+ * @param <V> type of the abstract value that is tracked
+ * @param <S> the store type used in the analysis
```

Comment:
ConditionalTransferResult below uses
` * @param <S> the {@link Store} used to keep track of intermediate results`
Maybe also make all these descriptions consistent?
In this particular case, the two descriptions aren't symmetric, which makes them a bit ugly.